react/MuiCozyTheme/ExpansionPanel/theme.js

```diff
@@ -0,0 +1,23 @@
+const radiusValue = '6px'
+const borderRadius = {
+  borderTopLeftRadius: radiusValue,
+  borderTopRightRadius: radiusValue,
+  borderBottomLeftRadius: radiusValue,
+  borderBottomRightRadius: radiusValue
+}
+
+const MuiExpansionPanel = {
+  root: {
+    ...borderRadius,
+    boxShadow: '0 4px 12px 0 rgba(0, 0, 0, 0.08)',
```

Comment:
Same here. Shouldn't we define our list of shadows in the appropriate property at the root of the theme instead of individual propoerties in each components?

Reply:
Definitely yes.

Reply:
Actually there are two problems. The first is that I can't access the `theme.shadows` variable from here.

By default, the `ExpansionPanel` is using the first elevation level (see https://github.com/mui-org/material-ui/blob/master/packages/material-ui/src/ExpansionPanel/ExpansionPanel.js#L163). That means that we have to either redefined the elevation shadows scale (that's a scale on 25 levels, see https://github.com/mui-org/material-ui/blob/master/packages/material-ui/src/styles/shadows.js) or not set the `boxShadow` generically and leave it to the app (I would use `withStyles` in Banks to provide the custom `boxShadow`, for example).

What do you think ?